cypress/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "env": {
+    "jest": true
+  },
+  "globals": {
+    "cy": true,
+    "context": true
+  }
+}
```

Comment:
This should be named .eslintrc.json, using eslintrc file without extension is deprecated.